components/infobox/wikis/rocketleague/infobox_league_custom.lua

```diff
@@ -176,6 +176,10 @@ function RLLeague:defineCustomPageVariables(args)
 end
 
 function RLLeague:addToLpdb(lpdbData, args)
+	if not String.isEmpty(args.liquipediatiertype) then
+		lpdbData['liquipediatier'] = args.liquipediatiertype or args.liquipediatier
```

Comment:
if we get to line 180 `args.liquipediatiertype` is never empty, so the ` or args.liquipediatier` is irrelevant

Reply:
You're right, fixed